modules/accredited_representative_portal/spec/requests/accredited_representative_portal/v0/power_of_attorney_requests_spec.rb

```diff
@@ -22,6 +21,9 @@
   end
 
   before do
+    AccreditedRepresentativePortal::PowerOfAttorneyForm.delete_all
+    AccreditedRepresentativePortal::PowerOfAttorneyRequestResolution.delete_all
+    AccreditedRepresentativePortal::PowerOfAttorneyRequest.delete_all
```

Comment:
We shouldn't need these. I think we make our test suite run with [transaction-oriented database cleaning on a per-spec-example basis](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/database_cleaner.rb#L26) and I think the `let!` is defeating that idea. Kind of just guessing, but I tried changing to `let` and causing the creation at the top of each spec example and I didn't need the `delete_all`'s. There are very few `delete_all`'s in the spec suite. For now, let's use `let` and cause the creation at the top of each example.